docs/make.jl

```diff
@@ -92,16 +92,19 @@ DocMeta.setdocmeta!(TrixiParticles, :DocTestSetup, :(using TrixiParticles); recu
 makedocs(sitename="TrixiParticles.jl",
          # Run doctests and check docs for the following modules
          modules=[TrixiParticles],
+         format=Documenter.HTML(),
```

Comment:
Does that change anything now? Or is it the default anyway?

Reply:
Yes this is just the default HTML output.